mock/mock.spec

```diff
@@ -213,13 +216,15 @@ install -d %{buildroot}/var/cache/mock
 mkdir -p %{buildroot}%{_pkgdocdir}
 install -p -m 0644 docs/site-defaults.cfg %{buildroot}%{_pkgdocdir}
 
+mkdir -p %{buildroot}%{_sysusersdir}
+install -p -D -m 0644 mock.conf %{buildroot}%{_sysusersdir}
+
 sed -i 's/^_MOCK_NVR = None$/_MOCK_NVR = "%name-%version-%release"/' \
     %{buildroot}%{_libexecdir}/mock/mock
 
 %pre filesystem
-# check for existence of mock group, create it if not found
-getent group mock > /dev/null || groupadd -f -g %mockgid -r mock
-exit 0
+echo "je tu nejaky skript nebo ne"
```

Comment:
Very likely there is:
```
rpm --eval "%sysusers_create_compat ./mock-core-configs/mock.conf"

# generated from mock.conf
getent group 'mock' >/dev/null || groupadd -f -g '135' -r 'mock' || : 
```

Reply:
:] I hope this do not get in final version.